node/src/chain_spec.rs

```diff
@@ -232,6 +233,15 @@ pub fn testnet_genesis(
 	para_id: ParaId,
 	chain_id: u64,
 ) -> GenesisConfig {
+	// This is supposed the be the simplest bytecode to revert without returning any data.
+	// We will pre-deploy it under all of our precompiles to ensure they can be called from
+	// within contracts. TODO We should have a test to ensure this is the right bytecode.
+	// (PUSH1 0x00 PUSH1 0x00 REVERT)
+	let revert_bytecode = vec![0x60, 0x00, 0x60, 0x00, 0xFD];
+	// TODO consider whether this should be imported from moonbeam precompiles
+	let precompile_addresses = vec![1, 2, 3, 4, 5, 6, 7, 8, 255, 256, 511]
```

Comment:
No longer true with the changes from https://github.com/PureStake/moonbeam/commit/7c4630c6a8ad163cc4a32768476cf44e19994e32 right?